tests/integration/retryableData.test.ts

```diff
@@ -34,6 +34,7 @@ import { isL2NetworkWithCustomFeeToken } from './custom-fee-token/customFeeToken
 describe('RevertData', () => {
   beforeEach('skipIfMainnet', async function () {
     await skipIfMainnet(this)
+    await skipIfNon18Decimals(this)
```

Comment:
This test doesn't work for non 18 decimals. The data we check against is hardcoded and doesn't match.

Safe to omit and only check for 18 decimal chains?